_posts/2022/2022-10-17-time-ranges-are-clearer-when-querying-databases.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Time Ranges are Clearer When Querying Databases"
+description: "Avoid text input to Active Record queries where you can"
+layout: article
+category: ruby
+image:
+  base: "2022/time-ranges-are-clearer-when-querying-databases"
+  alt: "Alarm Clock on pink and blue"
+  credit: "Icons8 Team"
+  source: "https://unsplash.com/photos/dhZtNlvNE8M"
+---
+
+Active Record‘s query interface has _many_ different ways to turn your logic into SQL to get rows from your database.
```

Comment:
Not clear why you'd need to "turn your logic into SQL".

Reply:
Ooh — now I'm at the examples I know what you mean: so that you can copy and paste into your DB, right?